Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY: all
+all: build
+
+.PHONY: build
+build:
+	go build ./...
+
+.PHONY: test
+test:
+	go test /...
+
+.PHONY: test-ci
+test-ci:
+	go test /... -covermode=atomic -coverpkg=./... -coverprofile=./coverage.txt -json | tee output.txt
+
+.PHONY: lint
+lint:
+	golangci-lint run $(addsuffix /...)
```

Comment:
What is `addsuffix`?

Reply:
Just removed, it is a leftover from the original setup and I personally don't know what it does.